fabric/src/main/java/com/discordsrv/fabric/console/executor/FabricCommandFeedbackExecutor.java

```diff
@@ -42,29 +39,29 @@ public FabricCommandFeedbackExecutor(MinecraftServer server, Consumer<Component>
         this.componentConsumer = componentConsumer;
     }
 
-    public ServerCommandSource getCommandSource() {
-        ServerWorld serverWorld = server.getOverworld();
+    public CommandSourceStack getCommandSource() {
+        ServerLevel level = server.overworld();
         //? if minecraft: <1.19 {
         /*Text text = Text.of("DiscordSRV");
         *///?} else {
-        Text text = Text.literal("DiscordSRV");
+        net.minecraft.network.chat.Component text = net.minecraft.network.chat.Component.literal("DiscordSRV");
          //?}
         //? if minecraft: >=1.21.9 {
-        Vec3d spawnPos = serverWorld == null ? Vec3d.ZERO : serverWorld.getSpawnPoint().getPos().toCenterPos();
+        Vec3 spawnPos = level == null ? Vec3.ZERO : level.getRespawnData().pos().getCenter();
         //?} else {
-        /*Vec3d spawnPos = serverWorld == null ? Vec3d.ZERO : Vec3d.of(serverWorld.getSpawnPos());
+        /*Vec3 spawnPos = Vec3.atLowerCornerOf(level.getSharedSpawnPos());
         *///?}
-        return new ServerCommandSource(
-                this, spawnPos, Vec2f.ZERO, serverWorld, 4, "DiscordSRV", text, server, null
+        return new CommandSourceStack(
+                this, spawnPos, Vec2.ZERO, level, 4, "DiscordSRV", text, server, null
         );
     }
     @Override
     //? if minecraft: <1.19 {
     /*public void sendSystemMessage(Text message, UUID sender) {
     *///?} else {
-    public void sendMessage(Text message) {
+    public void sendSystemMessage(net.minecraft.network.chat.Component message) {
      //?}
-            accept(Component.text(Formatting.strip(message.getString())));
+            accept(Component.text(ChatFormatting.stripFormatting(message.getString())));
```

Comment:
Side note but this should convert the Component to Adventure, as colors may be translated all the way to Discord